ACPs/77-reinventing-subnets/README.md

```diff
@@ -42,134 +44,133 @@ For messages produced by the P-Chain for a given Subnet, only that Subnet's vali
 
 The following Warp message payloads are introduced on the P-Chain:
 
-- `SubnetConversionMessage`
-- `RegisterSubnetValidatorMessage`
-- `SubnetValidatorRegistrationMessage`
-- `SubnetValidatorWeightMessage`
+- `L1ConversionMessage`
+- `RegisterL1ValidatorMessage`
+- `L1ValidatorRegistrationMessage`
+- `L1ValidatorWeightMessage`
 
 The method of requesting signatures for these messages is left unspecified. A viable option for supporting this functionality is laid out in [ACP-118](../118-warp-signature-request/README.md) with the `SignatureRequest` message.
 
 All node IDs contained within the message specifications are represented as variable length arrays such that they can support new node IDs types should the P-Chain add support for them in the future.
 
 The serialization of each of these messages is as follows.
 
-#### `SubnetConversionMessage`
+#### `L1ConversionMessage`
 
-The P-Chain can produce a `SubnetConversionMessage` for consumers (i.e. validator managers) to be aware of the initial validator set.
+The P-Chain can produce a `L1ConversionMessage` for consumers (i.e. validator managers) to be aware of the initial validator set.
 
 The following serialization is defined as a `ValidatorData`:
 
-| Field | Type | Size |
-| -: | -: | -: |
-| `nodeID` | `[]byte` | 4 + len(`nodeID`) bytes |
-| `blsPublicKey` | `[48]byte` | 48 bytes |
-| `weight` | `uint64` | 8 bytes |
-| | | 60 + len(`nodeID`) bytes |
-
+|          Field |       Type |                     Size |
+| -------------: | ---------: | -----------------------: |
+|       `nodeID` |   `[]byte` |  4 + len(`nodeID`) bytes |
+| `blsPublicKey` | `[48]byte` |                 48 bytes |
+|       `weight` |   `uint64` |                  8 bytes |
+|                |            | 60 + len(`nodeID`) bytes |
 
 The following serialization is defined as the `SubnetConversionData`:
```

Comment:
Should this be renamed as well? `ConversionData`?

Reply:
ye